### PR TITLE
bumping web5-js and tbdex-ks versions. 

### LINKED ISSUE #20: Update to reflect changes made to sdks

### PR DESCRIPTION
**Pull request by jiyoontbd**
closes #20

### CODE REVIEW

**Review by michaelneale — APPROVED** [review]
nice! Note that I don't think there is CI test coverage for this, but looks much nicer to my eyes.

File: src/client-test.ts [hunk 1]
```diff
@@ -41,16 +42,18 @@ const alice = await createOrLoadDid('alice.json')
...
+  issuer  : issuer.did,
   subject : alice.did,
   data    : {
     'beep': 'boop'
   }
 })
+const vcJwt = await vc.sign({ did: issuer })
 
 //
+console.log('vcJwt', vcJwt)
```
Comment (nitro-neal) [inline src/client-test.ts:56]: maybe want to remove the log here

File: src/client-test.ts [hunk 2]
```diff
@@ -69,14 +72,14 @@ const rfq = Rfq.create({
...
-    claims: [signedCredential]
+    claims: [vcJwt]
   }
 })
 
 await rfq.sign(alice)
 
-const rasp = await TbdexHttpClient.sendMessage({ message: rfq })
-console.log('send rfq response', JSON.stringify(rasp, null, 2))
+console.log('PFI DID', PFI_DID)
```
Comment (nitro-neal) [inline src/client-test.ts:81]: remove log (unless this is by design)

File: src/example/request-for-quote.ts [hunk 3]
```diff
@@ -21,7 +22,7 @@ if (!signedCredential) {
 //  Connect to the PFI and get the list of offerings (offerings are resources - anyone can ask for them)
 //
 const [ offering ] = await TbdexHttpClient.getOfferings({ pfiDid: pfiDid })
-console.log('offering:', JSON.stringify(offering, null, 2))
+// console.log('offering:', JSON.stringify(offering, null, 2))
```
Comment (nitro-neal) [inline src/example/request-for-quote.ts:25]:
```suggestion
 console.log('offering:', JSON.stringify(offering, null, 2))
```

**Review by nitro-neal — APPROVED** [review]
Awesome! Good stuff

File: src/config.ts [hunk 4]
```diff
@@ -31,19 +30,19 @@ export const config: Config = {
...
   },
-  did: undefined,
+  pfiDid: undefined,
   allowlist: JSON.parse(process.env['SEC_ALLOWLISTED_DIDS'] || '[]')
 }
 
 // create ephemeral PFI did if one wasn't provided. Note: this DID and associated keys aren't persisted!
 // a new one will be generated every time the process starts.
-if (!config.did) {
+if (!config.pfiDid) {
```
[the remark this answers is not in the document]
Comment (jiyoontbd) [reply src/config.ts:39]: yea it is a bit wonky. it is now using createOrLoadDid() method for PFI DID the same way we do it for alice and issuer DID. i still left the condition in as a failsafe. could probably be tweaked further in the future

**Pull request merged**